docker.go

```diff
@@ -34,12 +34,25 @@ func commandStartLocal(requiredOptions []string) {
 
 func commandStopLocal(requiredOptions []string) {
 	commandStopLocalContainer(gammaHandlerOptions(), requiredOptions)
-	commandStopLocalContainer(prismHandlerOptions(), requiredOptions)
+
+	if prismEnabled() {
```

Comment:
Not sure it's necessary. Nobody is going to call `stop-local` with `-no-ui` flag.